### ISSUE TITLE
docset out of date and script issues

### ISSUE THREAD

**Issue by tmclaugh** [pdhodgkinson/AWSCloudFormationTemplateReference-dash-docset#22]
I went to update the docset and had issues with out of date dependencies and Node version. I ended up doing a rewrite which can be found here.

https://github.com/ServerlessOpsIO/dash-docset-aws-cloudformation

If you don't intend to update the docset do you mind if I contact Kapeli to make mine the new one?

**Comment by pdhodgkinson** [comment]
@tmclaugh Yes, I will mark this as deprecated and turn it over to you. Thanks for taking this up

**Comment by tmclaugh** [comment]
Thank you! Wrapping up some minor things to do a new release and I'll submit the new one to Kapeli.
